CMakeLists.txt

```diff
@@ -517,9 +523,19 @@ if (HB_BUILD_SUBSET)
   add_dependencies(harfbuzz-subset harfbuzz)
   target_link_libraries(harfbuzz-subset harfbuzz ${THIRD_PARTY_LIBS})
 
-  if (BUILD_SHARED_LIBS)
-    set_target_properties(harfbuzz harfbuzz-subset PROPERTIES VISIBILITY_INLINES_HIDDEN TRUE)
-  endif ()
+  set_target_properties(harfbuzz harfbuzz-subset PROPERTIES 
+    VISIBILITY_INLINES_HIDDEN TRUE
+    FRAMEWORK TRUE
+    FRAMEWORK_VERSION "${HB_VERSION}"
+    PUBLIC_HEADER "${project_headers}"
+    PRODUCT_BUNDLE_IDENTIFIER "harfbuzz.harfbuzz-subset"
+    XCODE_ATTRIBUTE_INSTALL_PATH "@rpath"
+    OUTPUT_NAME "harfbuzz-subset"
+    XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY ""
+    MACOSX_FRAMEWORK_IDENTIFIER "harfbuzz-subset"
+    MACOSX_FRAMEWORK_SHORT_VERSION_STRING "${HB_VERSION}"
+    MACOSX_FRAMEWORK_BUNDLE_VERSION "${HB_VERSION}"
+  )
```

Comment:
Why is `harfbuzz-subset` now built without checking for `BUILD_SHARED_LIBS`, unlike all other libraries?

The comment about framework build also applies here.